byzcoin/service.go

```diff
@@ -2804,8 +2806,10 @@ func (s *Service) startChain(genesisID skipchain.SkipBlockID) error {
 	if err != nil {
 		return xerrors.Errorf("getting initial duration: %v", err)
 	}
-	s.viewChangeMan.add(s.sendViewChangeReq, s.sendNewView, s.isLeader, string(genesisID))
-	s.viewChangeMan.start(s.ServerIdentity().ID, genesisID, initialDur, s.getFaultThreshold(genesisID))
+	s.viewChangeMan.add(s.sendViewChangeReq, s.sendNewView, s.isLeader,
+		string(genesisID))
+	s.viewChangeMan.start(s.ServerIdentity().ID, genesisID, initialDur,
+		s.getFaultThreshold(latest.Hash))
```

Comment:
This was the line where it took the `f` from the genesis-block instead of the latest block.